astrocut/cutout_processing.py

```diff
@@ -371,12 +371,14 @@ def center_on_path(path, size, cutout_fles, target=None, img_wcs=None,
     response : str
         The file path for the output target pixel file.
     """
+    # Replace any slashes/spaces with % for filename conventions
+    target = target.replace('/', '%').replace(' ', '_')
```

Comment:
1. target defaults to `None` which in that case will lead to an exception `AttributeError: 'NoneType' object has no attribute 'replace'`

we'll want, at the very least: 

```suggestion
    if target:
        target = target.replace('/', '%').replace(' ', '_')
```

2. Curious how you decided on which characters to use here for replacement?

3. I think we actually want to use the correct target name in the header (line 395).  I think this string replacement should only be done for the file name, around line 427.

Reply:
The characters were chosen based on conventions I've seen before, where underscores denote separation between words etc. But essentially it is arbitrary. I've updated where the characters get replaced so that `OBJECT` keyword has the original format of the target name: 
<img width="773" alt="image" src="https://user-images.githubusercontent.com/32107699/158411503-598bea7b-610a-4fd8-80c3-555b86146de5.png">